core/src/test/java/org/elasticsearch/index/search/geo/GeoPointParsingTests.java

```diff
@@ -198,4 +200,10 @@ public static void assertCloseTo(final GeoPoint point, final double lat, final d
         assertEquals(point.lat(), lat, TOLERANCE);
         assertEquals(point.lon(), lon, TOLERANCE);
     }
+
+    @Override
+    protected XContent xContentForParser(BytesReference data) {
```

Comment:
I have a feeling I'm going to have a few more iterations of this as I do more centralizing. I'll stick with what I have for now and keep an open mind to that as I go. I'm not sure that we need the flexibility but the specificity might be nice....